superset-frontend/src/components/Menu/Menu.tsx

```diff
@@ -40,6 +40,9 @@ interface NavBarProps {
   bug_report_url?: string;
   version_string?: string;
```

Comment:
using the version_string would have prevented us from making a modification to superset (fork).

did you consider using that field and maybe adding a our build number information to it?

Reply:
Yes, I did consider using this field, that was my plan originally. The problem is makes the string long, it wraps on 2 lines and the 2 lines overlap each other.

Reply:
we should propose this addition to them. If the build number is not set then the UI could just not display it.

Reply:
I agree. I like this solution and if it can get merged upstream, that will be very convenient. Alternatively, if we really want to avoid modifications, could we fix the UI so the text wrapping works?

Reply:
I wanted to avoid UI/CSS modifications, this is why I took this route.

Reply:
Alright, I changed my code for an upstream contribution. I  added a try catch block because getting a variable from the config when it's missing throws an exception. First contribution ever and some pieces in Python!